app/src/main/java/io/gnosis/safe/ui/base/BaseDialog.kt

```diff
@@ -0,0 +1,5 @@
+package io.gnosis.safe.ui.base
+
+import androidx.fragment.app.DialogFragment
+
+abstract class BaseDialog : DialogFragment()
```

Comment:
Are we planning to add.stuff here?

Reply:
I think Vitaly has something here in his branch. Tomorrow we will merge stuff and sync. I will probably end up getting rid of this anyhow.